packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/ExecuteRulesTests.java

```diff
@@ -50,47 +63,265 @@
 @Test(groups = { TestGroup.RULES})
 public class ExecuteRulesTests extends RestTest
 {
+
     private UserModel user;
     private SiteModel site;
-    private FolderModel ruleFolder;
-    private FileModel file;
+    private FolderModel parentFolder;
+    private FolderModel childFolder;
+    private FileModel parentFolderFile;
+    private FileModel childFolderFile;
+    private RestRuleModel parentFolderRule;
+    private RestRuleModel childFolderRule;
 
     @BeforeClass(alwaysRun = true)
     public void dataPreparation()
     {
-        STEP("Create a user, site, folder and file in it");
+        STEP("Create user and a site");
         user = dataUser.createRandomTestUser();
         site = dataSite.usingUser(user).createPublicRandomSite();
-        ruleFolder = dataContent.usingUser(user).usingSite(site).createFolder();
-        file = dataContent.usingUser(user).usingResource(ruleFolder).createContent(CMISUtil.DocumentType.TEXT_PLAIN);
+    }
+
+    @BeforeMethod
+    public void setUp()
+    {
+        STEP("Create parent folder, rule folder and file in it");
+        parentFolder = dataContent.usingUser(user).usingSite(site).createFolder();
+        childFolder = dataContent.usingUser(user).usingResource(parentFolder).createFolder();
+        parentFolderFile = dataContent.usingUser(user).usingResource(parentFolder).createContent(CMISUtil.DocumentType.TEXT_PLAIN);
+        childFolderFile = dataContent.usingUser(user).usingResource(childFolder).createContent(CMISUtil.DocumentType.TEXT_PLAIN);
+
+        STEP("Create rules for parent and rule folders");
+        RestActionBodyExecTemplateModel addLockableAspectAction = createAddAspectAction(LOCKABLE_ASPECT);
+        RestRuleModel ruleModel = createRuleModel(RULE_NAME_DEFAULT, List.of(addLockableAspectAction));
+        ruleModel.setIsInheritable(true);
+        parentFolderRule = restClient.authenticateUser(user).withCoreAPI().usingNode(parentFolder).usingDefaultRuleSet().createSingleRule(ruleModel);
+        childFolderRule = restClient.authenticateUser(user).withCoreAPI().usingNode(childFolder).usingDefaultRuleSet().createSingleRule(createRuleModelWithDefaultValues());
+    }
+
+    /**
+     * Execute one rule with one action trying to add audio aspect to a file.
+     */
+    @Test(groups = { TestGroup.REST_API, TestGroup.RULES, TestGroup.ACTIONS, TestGroup.SANITY })
+    public void executeRules_onlyOwnedRules()
+    {
+        STEP("Check if file aspects don't contain Audio and Lockable ones");
```

Comment:
I don't think you're doing anything with LOCKABLE now.  There's also the import above.

Reply:
Oh - actually just update this comment. The next test uses Lockable.